soc/arm/nordic_nrf/nrf53/Kconfig.soc

```diff
@@ -98,6 +98,32 @@ config SOC_DCDC_NRF53X_HV
 	help
 	  Enable nRF53 series System on Chip High Voltage DC/DC converter.
 
+config NRF_SPU_FLASH_REGION_SIZE
+	hex
+	default 0x4000
+	help
+	  FLASH region size for the NRF_SPU peripheral
+
+config NRF_SPU_FLASH_START_ADDR_ALIGNMENT
```

Comment:
Why not `NRF_SPU_FLASH_REGION_ALIGNMENT`, actually?

Reply:
good question changed it